tests/diag.sh

```diff
@@ -2567,7 +2692,13 @@ case $1 in
 		fi
 		;;
    'kill-immediate') # kill rsyslog unconditionally
-		kill -9 $(cat $RSYSLOG_PIDBASE.pid)
+		command -v fuser > /dev/null 2>&1
```

Comment:
I think it would make sense to put this common code into a function.

Reply:
Hi @rgerhards - I agree that a function makes sense here, and will push that change shortly.

Reply:
I have pushed this change, with one more fix for the non-priv drop code path mentioned pending.